Lib9c/Model/Arena/ArenaInformation.cs

```diff
@@ -38,6 +38,10 @@ public ArenaInformation(List serialized)
             Ticket = (Integer)serialized[3];
             TicketResetCount = (Integer)serialized[4];
             PurchasedTicketCount = (Integer)serialized[5];
+            if (serialized.Count > 6)
+            {
+                PurchasedTicketCountDuringInterval = (Integer) serialized[6];
+            }
```

Comment:
serialized dictionary always contains interval field
```suggestion
            PurchasedTicketCountDuringInterval = (Integer) serialized[6];
```